packages/cli/src/models/network/TransactionController.ts

```diff
@@ -1,8 +1,12 @@
-import { Transactions, Logger, ZWeb3, TxParams } from 'zos-lib';
+import { Transactions, Logger, ZWeb3, TxParams, ABI } from 'zos-lib';
 import { isValidUnit, prettifyTokenAmount, toWei, fromWei } from '../../utils/units';
 import { ERC20_PARTIAL_ABI } from '../../utils/constants';
 import { allPromisesOrError } from '../../utils/async';
+import LocalController from '../local/LocalController';
```

Comment:
I am pretty certain controllers should not use other controllers.  Idea behind Controllers concept is usually a layer which communicates in our case between scripts layers and models layer. I suggest we stick to this pattern. Thoughts @jcarpanelli @spalladino?

Reply:
I completely agree with you. As a matter of fact, it was a pain for me to import the `LocalController` into this new `TransactionController` :cry:. That being said, it was in our tech debt plans to modularize both `NetworkController` and `LocalController` into different controllers and models as apropiate, and `getContractClass` is definitely a function that will live in a model. Btw, if you check `LocalController#L132` you will see that we have commented where this function should be defined (in a future cli `Contract` model)

Reply:
At the risk of being too much of a :policeman:, I'd suggest making that specific refactor _now_ (ie on this PR). I fear that if we allow this single reference between controllers, this will get out of hand real soon, and since this controller is using a single method from LocalController, it should be easy to refactor out to a separate model.

To be clear: I'm only saying to move `getContractClass` somewhere else to remove the need for this dependency, not to refactor the entire controllers.

Reply:
No more to say! :muscle: